packages/lime-system/files/etc/uci-defaults/92_add-lime-repos

```diff
@@ -10,28 +12,17 @@ feeds_file="/etc/opkg/limefeeds.conf"
   exit 0
 }
 
-ARCH="$OPENWRT_ARCH"
-[ -z "$ARCH" ] && { # compatibility with openwrt versions prior to 19.07
-    ARCH="$LEDE_ARCH"
-}
-
-
-[ -z "$ARCH" ] && {
-  echo "Release information not available, skipping opkg configuration"
-  exit 0
-}
-
 [ "$LIME_CODENAME" == "development" ] && {
-	base_url="http://snapshots.libremesh.org/packages/$ARCH";
-	key_name="7546f62c3d9f56b1"
-	key_content="RWR1RvYsPZ9WsTsmoVajHX9dyl2wL7grjcfFua1q8A99RWr2gF94lRdJ"
+	base_url="http://feed.libremesh.org/master";
+	key_name="a71b3c8285abd28b"
+	key_content="RWSnGzyChavSiyQ+vLk3x7F0NqcLa4kKyXCdriThMhO78ldHgxGljM/8"
 } || {
-	base_url="http://repo.libremesh.org/releases/$LIME_RELEASE/packages/$ARCH"
+	base_url="http://feed.libremesh.org/$LIME_RELEASE"
```

Comment:
there is no more need in providing a ARCH dependent URL because most of the packages are ARCH agnostic right?

Reply:
Yep, all the packages have `PKGARCH:=all` except `soporteremoto` (for no reason at all) and `tinc` (don't know why, anyway an other version of the same package is also present upstream: https://github.com/openwrt/packages/blob/master/net/tinc/Makefile).
Anyway this should be considered first in the Github Action on https://github.com/libremesh/lime-packages/blob/master/.github/workflows/build.yml

Reply:
Perfect thanks for the clarification

Reply:
As we are using the same key maybe we can just add they key to files/etc/opkg/keys instead ?

Reply:
Related https://github.com/libremesh/lime-packages/pull/777

Reply:
> As we are using the same key maybe we can just add they key to files/etc/opkg/keys instead ?

I moved the key variables out of the if block. I would not add it into files/etc/opkg/keys as I'm not sure it will not conflict with other OpenWrt packages... Do you know that this will not happen?

Reply:
I am not 100% sure that this will not happen.

Reply:
I just realized that neither the auto-usb-wwan have the PKGARCH specified.